frontend/src/component/pages/Home.scss

```diff
@@ -14,6 +15,41 @@
         margin-right: 20px;
       }
     }
+
+    .create-short-link-btn {
+      display: flex;
+      justify-content: center;
+    }
+  }
+
+  @include respond-to($phone) {
+    .title {
+      text-align: center;
+    }
+
+    .control.create-short-link {
+      flex-direction: column;
+      height: 180px;
+
+      .text-field-wrapper {
+        display: flex;
+        margin: 4px;
+
+        input {
+          width: 100%;
+        }
+      }
+
+      .create-short-link-btn {
+        flex: 1;
+        margin: 4px;
+        margin-top: 20px;
+
+        button {
```

Comment:
I wonder whether we can add  media query to `https://github.com/short-d/short/blob/master/frontend/src/component/ui/Button.scss` to style the Button.